src/curses/ui_call_flow.c

```diff
@@ -460,33 +462,45 @@ call_flow_draw_message(ui_t *ui, call_flow_arrow_t *arrow, int cline)
     timeval_to_time(msg_get_time(msg), msg_time);
 
     // Get Message method (include extra info)
-    sprintf(method, "%s", msg_method);
+    if (METHOD_MAXLEN <= (snprintf(method, METHOD_MAXLEN, "%s", msg_method))) {
+        method[METHOD_MAXLEN-1]='\0';
+    }
```

Comment:
What about initializing method with zero and snprintf respecting end of string character?

    memset(method, 0, sizeof(method));
    snprintf(method, METHOD_MAXLEN - 1, "%s", msg_method);

Reply:
Sure i can change it. I would suggest to allocate max_size+1 byte and use max_size inside snprintf. The  "- 1" is easy to forgot.
```
char method[METHOD_MAXLEN + 1]
memset(method, 0, sizeof(method));
snprintf(method, METHOD_MAXLEN, "%s", msg_method);
```

Reply:
Yep, I like that much more, also the code is cleaner (only arithmentical operation in variable declaration)